test/0140/uri-field-naming.test.js

```diff
@@ -0,0 +1,86 @@
+const { linterForAepRule } = require('../utils');
+require('../matchers');
+
+let linter;
+
+beforeAll(async () => {
+  linter = await linterForAepRule('0140', 'aep-140-uri-field-naming');
+  return linter;
+});
+
+test('aep-140-uri-field-naming should find warning', () => {
+  const oasDoc = {
+    openapi: '3.0.3',
+    paths: {
+      '/test1': {
+        get: {
+          responses: {
+            200: {
+              content: {
+                'application/json': {
+                  schema: {
+                    type: 'object',
+                    properties: {
+                      url: {
+                        type: 'string',
+                      },
```

Comment:
I recommend adding some other properties here that end in "url" but with some prefix.